2nd-gen/test-functions-mocha/README.md

```diff
@@ -0,0 +1,18 @@
+# Firebase Functions Test - Quickstart
+
+This quickstart demonstrates how to run unit tests on gen-2 functions.
```

Comment:
I keep asking myself if we're calling it "Functions v2" or "Functions 2nd gen", but looking at the [docs for the public preview](https://cloud.google.com/functions/docs/2nd-gen/overview) I think it's the later.

```suggestion
This quickstart demonstrates how to run unit tests on Cloud Functions (2nd gen).
```

Reply:
We're sticking with `v2` in Firebase-land